test/evergreen.yml

```diff
@@ -2490,6 +2521,29 @@ tasks:
             export "PATH=/opt/mongodbtoolchain/v3/bin:$PATH"
             ./workgen_perf_check.sh many-dhandle-stress.py output.log 50 500 500
 
+  - name: many-collection-test
+    commands:
+      - func: "fetch mongodb repo"
+      - func: "get project"
+      - func: "merge mongodb wt"
+      - func: "compile mongodb"
+        vars:
+          configure_env_vars:
+            PATH=/opt/mongodbtoolchain/v3/bin:$PATH
```

Comment:
Suggest moving these 2 lines to the build variant section.

Reply:
17651e2